### ISSUE TITLE
runseq for STOP_OPTION=nsteps is incorrect 

### ISSUE THREAD

**Issue by jedwards4b** [ESCOMP/CMEPS#255]
THE GLC_NCPL calculation used to compute the runseq is wrong when STOP_OPTION=nsteps.

**Comment by adamrher** [comment]
Does this problem go away when using SGLC? When I use SLGC (i.e., the new F2000) and run for 9 nsteps I get this error in the ESMF log files:

20220102 100336.566 ERROR            PET02 src/addon/NUOPC/src/NUOPC_Base.F90:950 Invalid argument  - setClock timeStep=10800s is not a divisor of runDuration=16200s 

When I change it to 6 nsteps, it runs to completion. Is this related to the issue in this thread?

**Comment by mvertens** [comment]
@adamrher - I think that you need to set ROF_NCPL to have the same time step as ATM_NCPL. Can you please point me to your sandbox.

**Comment by adamrher** [comment]
yes, setting ROF_NCPL=48 runs nstep=9 to completion. Here's my sandbox (on izumi): /home/aherring/src/cam6_3_040.arh

**Comment by mvertens** [comment]
@adamrher - the issue is that the model can only run on intervals of the longest coupling frequency. So if ROF_NCPL = 8 (the default) you can only run the model for 6, 12, 18... CAM time steps - assuming that ATM_NCPL=48. All the CAM tests set ROF_NCPL=CAM_NCPL so that you can run for any number of time steps that you want. Does that make sense?
